config/initializers/rpush.rb

```diff
@@ -0,0 +1,144 @@
+Rpush.configure do |config|
+  # Supported clients are :active_record and :redis
+  config.client = :redis
+
+  # Options passed to Redis.new
+  config.redis_options = { url: ENV["REDIS_RPUSH_URL"], driver: :ruby }
```

Comment:
shouldn't this fallback to `REDIS_URL` ? We use fallbacks everwhere, see `config/initializers/session_store.rb` for example

Reply:
@rhymes Nice catch! I noticed this elsewhere, but I totally missed it here. 🎉 

@joshpuetz Speaking of which, do you think we'll need to add another Redis instance for this? If so, we should [add Datadog instrumentation](https://github.com/forem/forem/blob/68831039d123ea33c11a36ac1aa12e92a09b93eb/app/lib/forem_stats_drivers/datadog_driver.rb#L14-L16) for it. Otherwise, we can piggyback off of the cache instance  we're currently using(we're only using about 50% of it) and revisit when we've got metrics.

Reply:
I added `REDIS_RPUSH_URL` in case we wanted to split it out to a separate redis instance, but I can't image we'd actually need to separate it out right now. Still, based on that file you linked to, I'll add a separate Datadog line for it! (Does that mean we have to do something on the Datadog side?)